client/src/reducers/projects.ts

```diff
@@ -152,6 +172,35 @@ export const projectsReducer = (
       };
     }
 
+    case PROJECT_STATS_LOADING: {
+      return {
+        ...state,
+        stats: {
+          ...state.stats,
+          [action.projectID]: {
+            allTimeReceived: 0,
+            allTimeUniqueContributors: 0,
+            roundStats: [],
+          },
+        },
+        error: undefined,
```

Comment:
I wouldn't set it to undefined otherwise if it failed loading the projects we can't know it cause error is undefined